DEVELOPING.md

```diff
@@ -354,6 +354,38 @@ stacktrace that was saved as a txt file.
 $ ndk-stack -sym obj/arm64-v8a -dump trace.txt
 ```
 
+## Release management
+
+The Mapbox Maps SDK for Android makes stable releases every 6 weeks. Between releases, updates are available as snapshots before the issuance of a beta build in week 2 and a release candidate in week 4, which is promoted to stable in Week 6.
+
+### What is a snapshot?
+A snapshot release is a mechanism by which the latest commits are available as a build. Android provides snapshots as unique builds. Snapshots are available continuously.
```

Comment:
Not sure what `Android provides snapshots as unique builds.` means

Reply:
+1, is this about us not supporting date/time driven snapshots?

Reply:
I don't have that much info, I copied it from our internal doc 😆 Will remove, I guess snapshots are well described in a section below.